HW_3/task_2.py

```diff
@@ -0,0 +1,30 @@
+# Требуется найти в массиве A самый близкий по величине элемент к заданному числу X. 
+# Если таких элементов несколько, вы вывести один любой. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+# Последняя строка содержит число X
+
+# *Пример:*
+
+# 5
+#     7 -2 3 5 1
+#     6
+#     -> 7
+
+import random
+count = 2147483647 #максимальный int для Python
```

Comment:
Можно сделать так
```suggestion
count = float('inf') #максимальный int для Python
```
Хотя это уже будет типом данных `float`, но это совершенно не важно. Работать будет